test/radio/radio-controller-test.ts

```diff
@@ -0,0 +1,81 @@
+import test from 'ava';
+import * as dgram from 'dgram';
```

Comment:
Is `dgram` a dependency that needs to be included in `package.json`?

Reply:
`dgram` is a core Node module so there's no need to include it as a dependency.